assets/main.css

```diff
@@ -12,4 +12,42 @@ body {
 .tester {
     color: red;
     font-size: 3em;
-}
\ No newline at end of file
+}
+
+
+/* Start RSB-2 */
```

Comment:
Should be a more descriptive name for what these styles are for e.g. "billing home page"